packages/vitest/src/node/pkg.ts

```diff
@@ -1,13 +1,16 @@
+import url from 'node:url'
 import c from 'picocolors'
 import { isPackageExists } from 'local-pkg'
 import { EXIT_CODE_RESTART } from '../constants'
 import { isCI } from '../utils/env'
 
+const __dirname = url.fileURLToPath(new URL('.', import.meta.url))
+
 export async function ensurePackageInstalled(
```

Comment:
I looked around and couldn't figure out where tests should go for this change, if anywhere. Would you like me to test this in some way?

Reply:
@sheremet-va will look into this.

Reply:
There are no tests for this feature.